driver/docker/docker.go

```diff
@@ -112,6 +115,14 @@ func Purge() error {
 		}
 	}
 
+	// get all volumes created by norma
+	_, err = cli.listVolumes()
+	if err != nil {
+		return err
+	}
+
+	// TODO: remove all volumes
```

Comment:
missing logic?

Reply:
Yes, there is a hiccup with volume management.
Will add this and request for approval.

Reply:
Added